internal/ports/convert.go

```diff
@@ -87,15 +90,20 @@ func (c *Client) Convert() *cobra.Command {
 				addressFamily = int32(metal.IPADDRESSADDRESSFAMILY__4)
 				public = true
 				addrs = append(addrs, metal.PortConvertLayer3InputRequestIpsInner{AddressFamily: &addressFamily, Public: &public})
+
+				addressType = "public"
 			}
 			if f := cmd.Flag("public-ipv6"); f.Changed {
 				addressFamily = int32(metal.IPADDRESSADDRESSFAMILY__6)
 				public = true
 				addrs = append(addrs, metal.PortConvertLayer3InputRequestIpsInner{AddressFamily: &addressFamily, Public: &public})
+
+				addressType = "public"
+				addressFamilyStrs = append(addressFamilyStrs, strconv.Itoa(int(metal.IPADDRESSADDRESSFAMILY__6)))
 			}
 
 			convToL3 := func(portID string) (*metal.Port, *http.Response, error) {
-				log.Printf("Converting port %s to layer-3 with addresses %v", portID, addrs)
+				log.Printf("Converting port %s to layer-3 with [%s] addresses IPv[%v]", portID, addressType, addressFamilyStrs)
```

Comment:
addressType will be `public` when both public and private (required, will always be true when enabling layer3).

Was the intent to say (pseudo-code):
`"Converting port %s to layer-3 with private IPv4" (conditionally "and public " (conditionally join(IPv4AndOrIPv6, ", ")) " addresses")`

Reply:
I don't think we offer this level of detail for anything else being created. We don't say, for example, "Creating a device in metro sv with plan c3.small.x86 and OS alpine"

I think we could omit this log message altogether if we really don't do it in the usual create or update case.
Perhaps a reason to keep it would be if the arguments to this command are so complicated that you never really know what to expect on the other end when you ask for something. I think we have a handle on that now that we are addressing the bonding and converting actions not performing the actions that they logically should have.

Reply:
Got it. Fixed